amm/interp/src/main/scala-2.13.2/ammonite/interp/MakeReporter.scala

```diff
@@ -0,0 +1,43 @@
+package ammonite.interp
```

Comment:
Seems adding that file is unnecessary.

Reply:
Pretty sure we need it, or we need to reconfigure the build to have a `2.13.1+/` folder